src/Maestro/Types/V1/Blocks.hs

```diff
@@ -0,0 +1,60 @@
+-- | Module to define types for /\"Blocks\"/ category endpoints defined at [docs.gomaestro.org](https://docs.gomaestro.org/docs/category/blocks).
+
+module Maestro.Types.V1.Blocks
+  ( BlockDetails (..)
+  , TimestampedBlockDetails (..)
+  ) where
+
+import           Data.Text               (Text)
+import           Data.Word               (Word64, Word32)
+import           Deriving.Aeson
+import           Maestro.Types.Common
+import           Maestro.Types.V1.Pools   (PoolId)
+import           Maestro.Types.V1.Common
+
+-- | Complete block Details when queried by its hash or height.
+data BlockDetails = BlockDetails
+  { _blockDetailsHash :: !BlockHash
+  -- ^ Block hash (identifier)
+  , _blockDetailsHeight :: !BlockHeight
+  -- ^ Block height
+  , _blockDetailsAbsoluteSlot :: !Word64
```

Comment:
We have a type, I believe it's called `SlotNo` for it.

Reply:
I just realized that maybe we can use https://github.com/maestro-org/haskell-sdk/blob/deb1e133cd7de989e9e85c2b44e9361f52e93776/src/Maestro/Types/Common.hs#L85-L88

Reply:
Yes, that's fine.